metrics/bertscore/bertscore.py

```diff
@@ -0,0 +1,131 @@
+It has been shown to correlate with human judgment on sentence-level and system-level evaluation. 
+Moreover, BERTScore computes precision, recall, and F1 measure, which can be useful for evaluating different language generation tasks.
+
+See the [README.md] file at https://github.com/Tiiiger/bert_score for more information.
+"""
+
+_KWARGS_DESCRIPTION = """
+BERTScore Metrics with the hashcode from a source against one or more references.
+
+Args:
+    `predictions` (list of str): prediction/candidate sentences
+    `refereces` (list of str or list of list of str): reference sentences
+    `lang` (str): language of the sentences; required (e.g. 'en')
+    `model_type` (str): bert specification, default using the suggested
+    model for the target langauge; has to specify at least one of
+    `model_type` or `lang`
+    `num_layers` (int): the layer of representation to use.
+    default using the number of layer tuned on WMT16 correlation data
+    `verbose` (bool): turn on intermediate status update
+    `idf` (bool or dict): use idf weighting, can also be a precomputed idf_dict
+    `device` (str): on which the contextual embedding model will be allocated on.
+    If this argument is None, the model lives on cuda:0 if cuda is available.
+    `nthreads` (int): number of threads
+    `batch_size` (int): bert score processing batch size
+    at least one of `model_type` or `lang`. `lang` needs to be
+    specified when `rescale_with_baseline` is True.
+    `rescale_with_baseline` (bool): rescale bertscore with pre-computed baseline
+Returns:
+    'precision': Precision,
+    'recall': Recall,
+    'f1', F1 score,
+    'hashcode': Hashcode of the library,
+"""
+
+class BERTScore(nlp.Metric):
+    def _info(self):
+        return nlp.MetricInfo(
+            description=_DESCRIPTION,
+            citation=_CITATION,
+            homepage="https://github.com/Tiiiger/bert_score",
+            inputs_description=_KWARGS_DESCRIPTION,
+            features=nlp.Features({
+                'predictions': nlp.Value('string', id='sequence'),
+                'references': nlp.Sequence(nlp.Value('string', id='sequence'), id='references'),
+            }),
+            codebase_urls=["https://github.com/Tiiiger/bert_score"],
+            reference_urls=["https://github.com/Tiiiger/bert_score",
+                            "https://arxiv.org/abs/1904.09675"]
+        )
+
+    def _compute(
+        self,
+        predictions,
+        references, 
+        lang=None,
+        model_type=None,
+        num_layers=None,
+        verbose=False,
+        idf=False,
+        device=None,
+        batch_size=64,
+        nthreads=4,
+        all_layers=False,
+        rescale_with_baseline=False,
+    ):
+        try:
+            import bert_score
+        except ImportError:
+            raise ImportError("Please install bert_score using 'pip install bert-score'.")
```

Comment:
You can have this import at the top of the script like a normal import.

When loading the script we are actually scanning the imports, testing them, and display an error message similar to the one you are throwing here if an import is not available (see [here](https://github.com/huggingface/nlp/blob/master/src/nlp/load.py#L284-L295))

Reply:
Awesome! I didn't notice that the ImportError has been handled in such a nice way.
I've clean it up in the new commit. I can help with adding the doc regarding BERTScore once some examples are added in the future.

Please feel free to let me know if there is something that needs to be changed.